libs/librepcb/projecteditor/schematiceditor/schematiceditor.cpp

```diff
@@ -696,31 +696,49 @@ void SchematicEditor::renameSchematic(int index) noexcept {
   }
 }
 
+QList<SI_Symbol*> SchematicEditor::getSearchCandidates() noexcept {
+  QList<SI_Symbol*> candidates = {};
+  foreach (const Schematic* schematic, mProject.getSchematics()) {
+    Q_ASSERT(schematic);
+    camdidates += schematic->getSymbols();
```

Comment:
Oooh damn, now there is a typo which makes CI fail :sob:

```suggestion
    candidates += schematic->getSymbols();
```